hw/ip/aes/dv/plan.md

```diff
@@ -0,0 +1,94 @@
+<!-- Copy this file to hw/ip/aes/dv/plan.md and make changes as needed. For
+convenience 'aes' in the document can be searched and replaced easily with the
+desired IP (with case sensitivity!). Also, use the testbench block diagram here:
+https://drive.google.com/open?id=1LfnTSutIW5E6zSCOCf4-scS8MQ8lXhPAPgSfFx2Aqh0
+as a starting point and modify it to reflect your aes testbench and save it
+to hw/ip/aes/dv/tb.svg. It should get linked and rendered under the block
+diagram section below. Once done, remove this comment before making a PR. -->
+
+{{% lowrisc-doc-hdr Foo DV Plan }}
```

Comment:
You should use Hugo compatible tag here. The failure happens at this line.

Reply:
Thanks @eunchan 
@rasmus-madsen could you re-generate this file and move it to aes/doc/dv_plan/index.md? We changed tool to use Hugo for generating document and some formats need to be changed. 

Before you push the change, run below command to see if any error.
cd opentitan
./util/build_docs.py --preview

If no error, you should see below message and you can open the link (http://localhost:1313/) to double check it.
> Serving pages from /usr/local/google/home/weicai/ot/opentitan/build/docs
> Web Server is available at http://localhost:1313/ (bind address 127.0.0.1)
> Press Ctrl+C to stop


@sriyerg seems like we need to template files for dv_plan. It's still at doc/dv_plan.md and need to move it to doc/dv_plan/index.md

Reply:
this file should be deleted... which as asked in
_Originally posted by @sriyerg in https://github.com/_render_node/MDIzOlB1bGxSZXF1ZXN0UmV2aWV3VGhyZWFkMjEwMTUyODM1OnYy/pull_request_review_threads/discussion_